d2bs/kolbot/libs/common/Pickit.js

```diff
@@ -264,7 +264,8 @@ MainLoop:
 			}
 
 			if (stats.useTk) {
-				Skill.cast(43, 0, item);
+				Skill.setSkill(43, 0);
```

Comment:
I think this slipped into the commit by mistake, but in any case it should respect the settings of packet casting

Reply:
fixed